easybuild/easyconfigs/m/MACS/MACS-1.4.2-1-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = "PythonPackage"
+
+name = 'MACS'
+version = '1.4.2-1'
```

Comment:
But you can't download directly from that URL, it seems like you need to use https://cloud.github.com/downloads/taoliu/MACS/MACS-1.4.2-1.tar.gz instead...